matrix-sdk-android/docs/modules.md

```diff
@@ -0,0 +1,18 @@
+# Module matrix-sdk-android
+
+## Welcome to the matrix-sdk-android documentation!
+
+This pages list the complete API that this SDK is exposing to a client application.
+
+*We are still building the documentation, so everything is not documented yet.*
+
+A few entry points:
+
+- **Matrix**: The app will have to create and manage a Matrix object.
```

Comment:
Could be nice to have link to the doc, but I did not find a way to make that work.

Reply:
@bmarty That may help you to add the correct references to classes: https://github.com/Kotlin/dokka/issues/555#issuecomment-1111195409

Reply:
Thanks, I will have a look on that!